pages/legal/disclaimers/disclaimers.md

```diff
@@ -1,8 +1,7 @@
 **Effective Date: July 26, 2018**
 
 The Blockstack Tokens are a crypto asset that is currently being developed by
-Blockstack Token LLC, a Delaware limited liability company, whose website can be
-found at [www.blockstack.com](https://www.blockstack.com). The website you are
-currently visiting [(www.blockstack.org)](www.blockstack.org) is sponsored by
-Blockstack PBC, an affiliate of Blockstack Token LLC, and should not be viewed
-as an offer or sale of securities.
+Blockstack Token LLC. The website you are currently visiting 
```

Comment:
Hi, @krtb still a bit too much trim.  We need to keep the `LLC, a Delaware limited liability company.` phrase.  Thanks for signing, almost there.

Reply:
Sorry about that @moxiegirl, I've added that line back in and pushed my changes. Thanks for your patience.